.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: php
+os: linux
```

Comment:
* Remove the travis.
* Run the following command to fix composer.json 
* `vendor/bin/console dev:composer:update-json-files -m SprykerEco.Heidelpay`.